akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -113,6 +113,7 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
     event match {
       case m: LogEventWithMarker ⇒
```

Comment:
Perhaps we should add a comment to LogEventWithMarker explaining `marker` may be `null`, as that is rather uncommon?

Reply:
Into the trait or at this line?

Reply:
In the trait would be the most clear IMHO

Reply:
Added this comment line into the trait

```
Marker attribute is nullable due to backward binary compatibility in the class `Warning4`
```